.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "printWidth": 120,
+  "tabWidth": 2,
+  "semi": false,
+  "singleQuote": true,
+  "trailingComma": "all",
+  "proseWrap": "always"
```

Comment:
I just switched my own behavior (actually last week 😄) to not introduce line-breaks in md or rst any more, since this becomes less readable e.g. on mobile devices and one can always activate soft-wrap on editor, so I would be in favor not to break, no strong opinion on this though.

Reply:
It doesnt change the rendered output though right? Could you play with these settings locally and let me know what do you prefer? https://prettier.io/docs/en/options.html#prose-wrap

Reply:
okay i removed this setting